.github/workflows/frontend-checks.yml

```diff
@@ -28,11 +28,3 @@ jobs:
       - name: Formatting
         if: always()
         run: npm run check-formatting
-
-      - name: Jest Tests
-        if: always()
-        run: npm run test
-
-      - name: Types
-        if: always()
-        run: npm run check-types
```

Comment:
why remove frontend tests?

Reply:
The front end is going to change significantly pretty soon so I didn't think it was that big of a priority. 

That and I didn't really understand the test that was failing at the time. It was something to do with the registration page... The page runs and completes registration successfully though.